.github/workflows/pr.yml

```diff
@@ -28,10 +28,7 @@ jobs:
       with:
         files: |
           plugins/**
-          tests/*.go
           tests/testdata/buf.build/**
-          tests/testdata/images/*.gz
-          Makefile
```

Comment:
At one point it made sense to rebuild everything if you changed common files. Now this is way too time consuming for ~200+ plugins. If we ever want to do something like this in the future, maybe we run a weekly full build of all plugins just as a sanity check.